packages/vscode-extension/src/common/Project.ts

```diff
@@ -111,6 +112,7 @@ export interface ProjectInterface {
 
   getDeviceSettings(): Promise<DeviceSettings>;
   updateDeviceSettings(deviceSettings: DeviceSettings): Promise<void>;
+  sendBiometricAuthorization(match: boolean): Promise<void>;
```

Comment:
I'd name the parameter `(is)successful`, match doesn't convey much meaning iMO.

Reply:
why? matching (face/finger) is exactly what you do. so it is a match or it is not. In fact when you fail to match you are not necessary unsuccessful because not matching strangers face is expected behaior